edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -38,6 +38,26 @@
         error: {
             status: gettext('Error'),
             message: gettext('An error has occurred during your onboarding exam. Please retry onboarding.')
+        },
+        other_course_approved: {
+            status: gettext('Approved in Another Course'),
+            message: gettext(
+                'You have completed an onboarding exam in another course, ' +
```

Comment:
I'm not sure whether you saw my feedback in the thread you posted, so I wanted to repost it here.

Nit: I wonder if "you have completed an onboarding exam in another course" may incorrectly suggest that a learner just needs to  complete an onboarding exam in order to be verified. Maybe "Your onboarding profile has been approved in another course" or something along those lines. Or "Your submitted onboarding attempt in another course has been approved"?

Reply:
Meant to update this piece based on your feedback, thank you for reminding me!